bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/EdCompTargetList.java

```diff
@@ -104,6 +104,8 @@ public void textBoxKeyPress(final TextBoxWidget tbwe) {
 
         _agsPub.subscribe((obs, oldOptions, newOptions) -> updateGuiding());
 
+        _curSelection = None.instance();
```

Comment:
Yes, this coupled with changing `_curSelection` from an `ImEither` to an `Option<ImEither>` and then using it in that way so that we never have to deal with `null`, which came up as a very rare thing.